Assets/MixedRealityToolkit/Utilities/Physics/TwoHandRotateLogic.cs

```diff
@@ -25,19 +25,19 @@ public class TwoHandRotateLogic
         /// <summary>
         /// Setup the rotation logic.
         /// </summary>
-        public void Setup(Dictionary<uint, Vector3> handsPressedMap, Transform t, RotationConstraintType rotationConstraint)
+        public void Setup(Vector3[] handsPressedArray, Transform t, RotationConstraintType rotationConstraint)
```

Comment:
>Vector3[] handsPressedArray [](start = 26, length = 27)

Not that this was defined well previously - it would be good to have a <param> doccomment about what the shape of this array is (i.e. it should be of length 2, where the new location is in the 1-th index, and the old location is in the 0-th index.

I think same goes for any public function in this class that takes a Vector3 array (I think they're all the same thing)